src/components/Bag/index.js

```diff
@@ -1,5 +1,6 @@
 import React, { useState, useEffect, useCallback } from "react";
 
+import { useQuery } from "react-query";
```

Comment:
Just making a note in case anybody else is seeing the same thing and getting confused - github is showing me 2 sets of changes:`src/components/Bag/*` as well as `src/components/bag/*`

`src/components/Bag/*` is showing minor changes
`src/components/bag/*` is showing everything as a new file

Doing a local checkout of this branch, I ended up with just `src/components/Bag/*`
😕

Reply:
Weird. I checked in VSCode after I renamed the folder and then when committing to check the rename 😕

Reply:
Damn it. I should have saved folder rename for another day!. I thought, now that am messing with bags, lets rename it 🤦🏽 😩 

Here is what I get when I rebasing this branch on master:
![image](https://user-images.githubusercontent.com/1250790/130643386-27daa5ff-78f0-4f94-82e4-f03a4f89bd1b.png)

Reply:
I double checked that the contents in `src/components/Bag/*` in the previous commit (`821fc8f`) match the contents in `src/components/bag/*` in the rename commit you mentioned (`c171e99`). They seem to match, save for the lowercase change in an import, so I think it's safe to delete if you want to go that route 👍 

```diff
❯ git --no-pager diff 821fc8f:./src/components/Bag/ c171e99:./src/components/bag/
diff --git a/downloads/index.js b/downloads/index.js
index 60eae255..3235dddd 100644
--- a/downloads/index.js
+++ b/downloads/index.js
@@ -20,7 +20,7 @@ import buildID from "components/utils/DebugIDUtil";
 import { useTranslation } from "i18n";
 
 import { getHost } from "components/utils/getHost";
-import constants from "components/Bag/constants";
+import constants from "components/bag/constants";
 
 const useStyles = makeStyles((theme) => ({
     closeButton: {
```

As for the rebase, without seeing what was being rebased and where it was in the rebase process, it's hard to tell what exactly happened. It looks like perhaps some of the changed files were left out of staging so the rebase step couldn't continue. If you want help retrying the rebase, I could give an extra pair of eyes, or if not, deleting the commit should work too 👍

Reply:
Sounds good. I thinking to delete this commit now, do the rebase and do the folder rename in separate PR. Let see what @psarando thinks as well.